ephios/static/ephios/js/main.js

```diff
@@ -1,4 +1,4 @@
-$(function () {
+$(document).ready(function () {
```

Comment:
Is there a semantic difference to `$()`?

Reply:
TIL no. I prefer the verbose variant anyway because this is what tutorials use. never heard of the short variant to date.